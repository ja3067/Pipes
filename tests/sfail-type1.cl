def foo(x : str, y : int):
	return x + y

x = 4.0
y = 4

foo(x, y)

