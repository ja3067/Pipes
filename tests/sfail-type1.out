STypeError: unsupported operand type(s) for binary +: 'string' and 'int'